### PR TITLE
[DOCS] Relocate Use Great Expectations with Google Cloud Platform and BigQuery

### PR DESCRIPTION
**Pull request by kwcanuck**
In this [Slack conversation](https://greatexpectationslabs.slack.com/archives/C03B8DZCJ07/p1694188146803399), @jcampbell requested that we move the content in [Use Great Expectations with Google Cloud Platform and BigQuery](https://docs.greatexpectations.io/docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery/) to [Connect to SQL database source data](https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data). This PR implements this change. 

[DOC-595](https://greatexpectations.atlassian.net/browse/DOC-595)

[DOC-595]: https://greatexpectations.atlassian.net/browse/DOC-595?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ import Prerequisites from '/docs/components/_prerequisites.jsx'
 import ImportGxAndInstantiateADataContext from '/docs/components/setup/data_context/_import_gx_and_instantiate_a_data_context.md'
 import AfterCreateSqlDatasource from '/docs/components/connect_to_data/next_steps/_after_create_sql_datasource.md'
 import PostgreSqlConfigureCredentialsInConfigVariablesYml from '/docs/components/setup/dependencies/_postgresql_configure_credentials_in_config_variables_yml.md'
+import Prerequisites from 'docs/deployment patterns/components/deployment_pattern_prerequisites.jsx'
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
```
Comment (cdkini) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md]: The error comes from importing Prerequisites again - its also defined on L10

Comment (cdkini) [reply docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md]: I believe deleting this line will resolve the Netlify build issue

Comment (kwcanuck) [reply docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/connect_sql_source_data.md]: @cdkini thanks for the recommendation. I missed the duplicate Prerequisites reference as the path appeared different. I've removed one of the references.

**Review by Rachel-Reverie — APPROVED** [review]
LGTM

**Pull request merged**